### ISSUE TITLE
Problem with unicode identifiers

### ISSUE THREAD

**Issue by ghazpar** [DRMacIver/hypothesis#209]
In my test functions using Python 3, I can't use a keyword argument that has unicode (non ascii) characters in its identifier. For example:
```
from hypothesis import given
from hypothesis.strategies import integers

@given(é=integers(min_value=0, max_value=5))
def test(é):
    print(é)

test()
```
I get a `ValueError: 'é' is not a valid python identifier` exception.

Is this unavoidable?

**Comment by DRMacIver** [comment]
Are you using the latest version? This should already be fixed in 1.12.0

**Comment by ghazpar** [comment]
No, I was running 1.10; problem solved.

Thanks!
